Issue56 add templating system

TabrisArg opened the pull request:
issue #56

* [X] Added Front Matter to index.html in order for it to make use of the home layout
* [X] Removed redundant HTML code at the top and bottom to make room for the header and footer respectively

andiechu commented on index2.html:
```diff
@@ -0,0 +1,211 @@
+---
```
Why there's index.html and index2.html? Since now we are not modified on top of master branch (the production enviornment), you can develop the jekyll templates and pages from a fresh new start without worrying about messing the old website. Just make sure everyone can pull the use-jekyll branch and start running it on jekyll.

TabrisArg replied:
File deleted

TabrisArg commented:
* [X] Fixed all the pages so they use Jekyll's layouts and display correctly
* [X] Added a "base" that can be used in every page and can be built upon in the future. The base layout contains only the header and footer wrappers to be reused across the website
